apns/test/abacustest.py

```diff
@@ -65,12 +77,25 @@ def bohrium_config(**kwargs):
 
     Returns:
         bohrium settings: dict, involved in abacustest configuration
+    
+    Note:
+        However, this way is highly not recommended, because the password is stored in plain text. For abacustest, 
+        there is an alternative way in which the account and password can be read from two environment variables:
+        ```
+        export BOHRIUM_USERNAME=<bohrium-email>
+        export BOHRIUM_PASSWORD=<bohrium-password>
+        export BOHRIUM_PROJECT_ID=<bohrium-project-id>
+        ```
```

Comment:
yes I think so, so I think it would be better to tell user to set envars in somewhere like a manual or hands-on. I will do this, thanks. But also the way to store personal account and pwd is really unsafe.